httpd/handler.go

```diff
@@ -667,6 +673,99 @@ func (h *Handler) serveProcessContinuousQueries(w http.ResponseWriter, r *http.R
 	w.WriteHeader(http.StatusAccepted)
 }
 
+func (h *Handler) serveRunMapper(w http.ResponseWriter, r *http.Request) {
+	// we always return a 200, even if there's an error because we always include an error object
+	// that can be passed on
+	w.Header().Add("content-type", "application/json")
+	w.WriteHeader(200)
```

Comment:
the `WriteHeader`? I guess that can be removed. I did want to have a comment in there that it's always 200 unless something catastrophic happens.